vendor/src/github.com/docker/engine-api/types/container/policy.go

```diff
@@ -0,0 +1,13 @@
+package container
+
```

Comment:
Yeah, I mostly put this there to introduce the type `PolicyType` for well-know policies based on our last discussion. Having a type makes it a little more concrete (and verifiable) than just string constants as keys.

The labels themselves will need more thought based on whether docker would use the well known policy labels (similar to mac-address, cpu-shares flag etc).